README.md

```diff
@@ -12,23 +12,42 @@ To build and install mgclient from source you will need:
    - CMake version >= 3.4
    - OpenSSL version >= 1.0.2
    - C compiler supporting C11
+   - C++ compiler supporting C++14
 
 Additionally, if you want to build tests, you'll need:
-   - C++ compiler supporting C++14
+   - [Google Test](https://github.com/google/googletest)
+
+To install minimum compile dependencies on Debian / Ubuntu:
+
+```
+apt-get install -y git cmake make gcc g++ libssl-dev
+```
+
+On RedHat / CentOS / Fedora:
+
+```
+yum install -y git cmake make gcc gcc-c++ openssl-devel
+```
+
+Once everything is in place, create a build directory inside the source
+directory and configure the build by running CMake from it:
+
+```
+mkdir build
+cd build
+```
 
-Once everything is place, create a build directory inside the source directory
-and configure the build by running CMake from it:
+If you do not want to build tests, pass `-DBUILD_TESTING=OFF` to CMake like
+this:
 
 ```
-$ mkdir build
-$ cd build
-$ cmake ..
+cmake -DBUILD_TESTING=OFF ..
 ```
 
-If you do not want to build tests, pass `-DBUILD_TESTING=OFF` to CMake like this:
+If you want to build tests, run default CMake:
 
 ```
-$ cmake -DBUILD_TESTING=OFF ..
+cmake ..
 ```
```

Comment:
I think that we should change that tests aren't built by default and remove the whole section that refers to building tests.

Reply:
It makes sense to me. I'm not sure what's the standard in the CMake world, but that would be easier for the average user.